src/Stratis.Bitcoin.Features.Consensus/ConsensusFeature.cs

```diff
@@ -203,7 +203,7 @@ public void RegisterRules(IConsensus consensus)
                     new WitnessCommitmentsRule(), // BIP141, BIP144
                     new BlockSizeRule(),
 
-                    new PosBlockContextRule(), // TODO: this rule needs to be implemented
+                    new PosBlockSignatureRepresentationRule(), // TODO: this rule needs to be implemented
```

Comment:
Is the comment to the right of this still the case ?

Reply:
No, this should have been removed. I'll fix it.